independent-projects/tools/devtools-testing/src/main/java/io/quarkus/devtools/testing/WrapperRunner.java

```diff
@@ -105,7 +107,7 @@ public static int run(Path projectDir, Wrapper wrapper) {
     private static String getMavenSettingsArg() {
         final String mavenSettings = System.getProperty("maven.settings");
         if (mavenSettings != null) {
-            return mavenSettings;
+            return Files.exists(Paths.get(mavenSettings)) ? mavenSettings : null;
```

Comment:
Minor nitpick: use `Path.of` instead :)

Reply:
Yeah it's a shortcut you are right. I think we can live with it :)